integrated_channels/canvas/models.py

```diff
@@ -101,6 +101,9 @@ def channel_code():
     def get_learner_data_transmitter(self):
         return CanvasLearnerTransmitter(self)
 
+    def get_subsection_learner_data_exporter(self, user):
+        return CanvasLearnerExporter(user, self)
```

Comment:
given this just returns the same thing as `get_learner_data_exporter` what is the point of defining this here? I don't see it specifically called (at least not elsewhere in this PR)

Reply:
😬 Good catch! This just straight up isn't needed. I had thought it would be needed early in development while I had it in my head that I would create a completely different assessment level exporter class rather than just add assessment level exporter capabilities to the existing learner data exporter. You can see now I'm just returning the learner data exporter [here](https://github.com/edx/edx-enterprise/blob/771db5b8391249e57313966e3f0432d4a6e24fe0/integrated_channels/integrated_channel/models.py#L178). Will remove 👍